extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -1256,8 +1267,9 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.fabricjsOptions[value] = color.rgbaString;
           onChangeFunc[value]();
         };
+        picker.onOpen();
         picker.setOptions({
-          color: ctrl.fabricjsOptions[value]
+          color: ctrl.fabricjsOptions[value],
```

Comment:
Hm, seems like we are passing a JSON here? Maybe we should avoid trailing commas (see "Trailing Commas in JSON" section [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)).

Reply:
Done.